extraDevfileEntries.yaml

```diff
@@ -1,15 +1,15 @@
 schemaVersion: 1.0.0
 samples:
   - name: nodejs-basic
-    displayName: Basic NodeJS
+    displayName: Basic Node.js
     description: A simple Hello World Node.js application
     icon: https://raw.githubusercontent.com/maysunfaisal/node-bulletin-board-2/main/nodejs-icon.png
     tags: ["NodeJS", "Express"]
     projectType: nodejs
     language: nodejs
     git:
       remotes:
-        origin: https://github.com/redhat-developer/devfile-sample.git
+        origin: https://github.com/nodeshift-starters/devfile-sample.git
```

Comment:
if you look at the original sample [devfile](https://github.com/redhat-developer/devfile-sample/blob/master/devfile.yaml#L16-L17), we have different endpoint ports because a devfile does not allow for the same port. Since this is dev preview in ODC, we hacked the prev sample to run on [3001](https://github.com/redhat-developer/devfile-sample/blob/master/src/Dockerfile#L8)

this way it can run on both odo and ODC

If you would like to test it out with
1. [odo](https://github.com/openshift/odo) - just create an odo component and do a push
2. [ODC](https://github.com/openshift/console) - build either the master/release-4.8 branch and launch a local ODC (Check [README](https://github.com/openshift/console#build-everything) on how to access ODC via locathost:9000) or launch an Openshift 4.8 cluster, and import via Devfile to paste the sample repo link

Reply:
@maysunfaisal It looks like the sample port has been updated. Can you retest?

Reply:
* Updated the buildguidance to use port 3001- https://github.com/nodeshift-starters/devfile-sample/commit/35b0146e8092a72df3835d0fe0a06e0f663b1b6e
* Set port 3001 in the Dockerfile via an environment variable - https://github.com/nodeshift-starters/devfile-sample/commit/93f82c90d4e3072e2f3616815b5c51eec8050cb8

I have tested importing the sample through the OpenShift UI, and it works as expected and I can access the running application. 

But, I am now hitting issues while testing the sample with odo CLI, it seems to be failing to push.

Reply:
@BethGriggs you're right. The odo component fails to come up due to the "placeholder" image we have and we have an issue for this in odo https://github.com/openshift/odo/issues/4318

since this is the current behaviour, this update looks good to me